src/processes/follow-topic-process.js

```diff
@@ -134,18 +134,30 @@ const followTopicProcess = async (job, done) => {
 
         await chat.create();
         const channelState = await chat.watch();
-
         if (channelState.messages.length === 0) {
           await chat.sendMessage({
             text: communityMessageFormat.message,
-            user_id: communityMessageFormat.user_id
+            user_id: communityMessageFormat.user_id,
+            is_auto_message: true
           });
 
           try {
             // await chat.stopWatching();
           } catch (error) {
             console.log('::: Error on stopWatching :::', error);
           }
+        } else if (
+          channelState.messages.length === 1 &&
+          channelState.messages[0]?.is_auto_message
+        ) {
+          // add new condition based on ticket ping-4227
+          // check if first message is auto message from sender
+          await chat.sendMessage({
+            text: communityMessageFormat.message,
+            user_id: communityMessageFormat.user_id
+          });
+        } else {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, there's no error handling for `chat.sendMessage()`. It's important to handle potential errors to prevent unexpected behavior or crashes.
```diff
+ try {
  await chat.sendMessage({
    text: communityMessageFormat.message,
    user_id: communityMessageFormat.user_id
  });
+ } catch (error) {
+   console.error('::: Error on sendMessage :::', error);
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->